contracts/contracts/buyback/Buyback.sol

```diff
@@ -24,7 +23,10 @@ contract Buyback is Governable {
     // Swap from OUSD
     IERC20 immutable ousd;
 
-    // Swap to OGN
+    // Swap to OGV
+    IERC20 immutable ogv;
+
+    // OGN for Uniswap path
```

Comment:
I don't think we need to swap through OGN, see later comment on the swap path. Since that's the case, we can remove the OGN contract variable?